internal/pkg/service/buffer/worker/service/conditions.go

```diff
@@ -141,6 +141,7 @@ func (c *checker) watchImportConditions(ctx context.Context, wg *sync.WaitGroup)
 				export := event.Value
 				if !c.assigner.MustCheckIsOwner(export.ReceiverKey.String()) {
 					// Another worker node handles the resource.
+					delete(c.importConditions, export.ExportKey)
```

Comment:
- kazdy worker zodpoveda za inu cast `receivers`, ktorym kontroluje `conditions`
- vid `c.assigner.MustCheckIsOwner(export.ReceiverKey.String())` v kode
- distribucia sa moze zmenit, napr. ak sa robi deploy, alebo sa prida dalsi worker
- v tom pripade tu chybalo vymazanie starych dat
- ono je to este chranene `task lock`, tj. aj ked tu bola chyba, tak vzdy bezal iba jeden task
- ale zbytocne to logovalo konflikty a zatazovalo service